### PR TITLE
Add Getting Started and Troubleshooting to README.

### PR DESCRIPTION
**Pull request by scallyw4g**
I had a hard time finding some of this information, so I thought I would
add it here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,31 @@ You might want to add the new ```.build_ano``` build output folder to your .giti
...
 <img src="docs/Toscanini.png" alt="Arturo Toscanini" height="150" width="147"/>
+
+## Getting Started
+
+There is an excellent [getting started guide for ino](http://inotool.org/quickstart) that contains
+everything you should need to get the Blink program running on your board.
+
+## Troubleshooting
+
+HALP!  I am getting:
```
Comment (scottdarch) [inline README.md:41]: HELP

File: README.md [hunk 2]
```diff
@@ -30,3 +30,31 @@ You might want to add the new ```.build_ano``` build output folder to your .giti
...
+```
+avrdude: stk500_recv(): programmer is not responding
+avrdude: stk500_getsync() attempt 1 of 10: not in sync: resp=0x00
+avrdude: stk500_recv(): programmer is not responding
+avrdude: stk500_getsync() attempt 2 of 10: not in sync: resp=0x00
+...
+```
+
+The ```-m``` flag is particularly important and your upload may fail with the above error if
```
Comment (scottdarch) [inline README.md]: use single backtick for `inline code`

File: README.md (hunk 1 above)
Comment (scallyw4g) [inline README.md:41]: I could change it if you'd prefer, that was my attempt at comedy :)

Comment (scottdarch) [inline README.md:41]: ah, no. levity is fine. Just fix the backtick formatting and I'll merge. Thanks.

Comment (scallyw4g) [inline README.md:41]: Fixed.

**Comment by scottdarch** [comment]
Thanks!

**Pull request merged**